cli/fmt_errors.rs

```diff
@@ -48,8 +48,9 @@ pub fn format_location(frame: &JsStackFrame) -> String {
     return cyan("native").to_string();
   }
   let mut result = String::new();
-  if let Some(file_name) = &frame.file_name {
-    result += &cyan(&format_file_name(file_name)).to_string();
+  let file_name = frame.file_name.clone().unwrap_or_default();
+  if !file_name.is_empty() {
+    result += &cyan(&format_file_name(&file_name)).to_string();
```

Comment:
Do we have a test case for empty file name?

Reply:
I noticed it happen in the repl when `reportError()` is called (`throw new Error()` doesn't show pretty errors in the repl). Should be part of the other PR really. I'll take it out.

Reply:
Ah that's true... Scripts compiled in REPL don't have a filename and I found no way to supply it (AFAIK Node also has this quirk)